_posts/2018-10-30-using-helm.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title:  "Using Helm to deploy the Strimzi Kafka Operator"
+date: 2018-10-31
+author: seglo
```

Comment:
You can add your self to this file to have the name replaced: https://github.com/strimzi/strimzi.github.io/blob/master/_data/authors.yml

Reply:
Thanks.  Done.